app/controllers/spin.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  spinService: Em.inject.service('spin'),
```

Comment:
Services can be created and injected ad-hoc into Ember's first-class citizens (controllers, components, routes, etc) as a dependency. There are also initializers that can be made available to an entire set of those first-class citizens instead of just one.